<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>theBall Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="8,2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8" stretch="9,1">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Information</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Noeuds</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Propriétés</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Nom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="node_name"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDoubleSpinBox" name="node_pos_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="node_pos_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="node_pos_z"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="node_rotation_z">
             <item>
              <widget class="QDoubleSpinBox" name="node_rotation_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="node_rotation_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_30"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="node_list"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QPushButton" name="node_add">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="node_delete">
            <property name="text">
             <string>Effacer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Lumiers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Propriétés</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="light_type">
             <item>
              <property name="text">
               <string>Ponctuelle</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Dirictionelle</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QDoubleSpinBox" name="light_pos_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_pos_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_pos_z"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Ambient</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_19">
             <item>
              <widget class="QDoubleSpinBox" name="light_ambiant_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_ambiant_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_ambiant_z"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Diffuse</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_20">
             <item>
              <widget class="QDoubleSpinBox" name="light_diffuse_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_diffuse_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_diffuse_z_2"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Spéculaire</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QDoubleSpinBox" name="light_specular_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_specular_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="light_specular_z"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Shininess</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="light_shininess"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="node_type"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Ambient de la scene</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QDoubleSpinBox" name="light_scene_x"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="light_scene_y"/>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="light_scene_z"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Environement</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Skybox</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="6,1">
             <item>
              <widget class="QLineEdit" name="skybox_front"/>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_front_browse">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="6,1">
             <item>
              <widget class="QLineEdit" name="skybox_back"/>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_back_browse">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="6,1">
             <item>
              <widget class="QLineEdit" name="skybox_top"/>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_top_browse">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="6,1">
             <item>
              <widget class="QLineEdit" name="skybox_bottom"/>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_bottom_browse">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="6,1">
             <item>
              <widget class="QLineEdit" name="skybox_left"/>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_left_browse">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="6,1">
             <item>
              <widget class="QLineEdit" name="skybox_right"/>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_right_browse">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="skybox_apply">
               <property name="text">
                <string>Apliquer</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="skybox_clear">
               <property name="text">
                <string>Effacer</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="skybox_enable">
             <property name="text">
              <string>Activer le Skybox</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Fog</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Couleur</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QDoubleSpinBox" name="fog_x"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="fog_y"/>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="fog_z"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Début</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="fog_start"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Fin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="fog_end"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="fog_enable">
             <property name="text">
              <string>Activer le FOG (brouaillard)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrire"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuFichier"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOuvrire">
   <property name="text">
    <string>Ouvrire</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="text">
    <string>Enregistrer sous...</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
